src/modules/roc_core/target_darwin/roc_core/format_tid.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2019 Roc authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <pthread.h>
+#include <stdint.h>
```

Comment:
should I include `<stdint.h>` here or include `core/stddefs.h`

Reply:
We usually use `core/stddefs.h`, but `stdint.h` is also OK here because we're inside target directory and can be sure that `stdint.h` is available.